bindings/java/Makefile

```diff
@@ -2,7 +2,9 @@ INCLUDE_DIRS = ../../src ../../blst/bindings
 
 LIBRARY_FOLDER=src/main/resources/ethereum/ckzg4844/lib
 
-CC_FLAGS=
+FIELD_ELEMENTS_PER_BLOB ?= 4096
+
+CC_FLAGS = -DFIELD_ELEMENTS_PER_BLOB=$(FIELD_ELEMENTS_PER_BLOB)
```

Comment:
Is it possible to add this in the command itself similar to src/Makefile? We override the CC_FLAGS in our CI to create the java library itself.